modules/ROOT/pages/introduction/resilience.adoc

```diff
@@ -0,0 +1,55 @@
+= Resilience
+:description: This section describes the resilience features of Ops Manager.
+
+Neo4j Ops Manager is equipped with multiple features to ensure resilient service.
+
+== Rate limiting
+To avoid the server being overloaded by one or a few clients, a rate limit is applied per IP address. This helps ensure that the server is always available to respond to requests by putting a limit on the load an individual connection is allowed to place on the system.
+
+The default configuration can be changed with the following server configuration parameters.
```

Comment:
Do you think we need to include these parameters in `pages/installation
/server.adoc` since it used to be the complete list? Or would the list be too long there?

Reply:
I think it's already quite long. We've never gotten any negative feedback or requests to adjust the default rate limiting configuration, just inquiries about the specific details. So in my mind, this is probably more for information than anything, although it is of course fully possible to customize the values too.